om4labs/catalogs/obs_catalog_gfdl.yml

```diff
@@ -3,6 +3,40 @@ plugins:
     - module: intake_xarray
 sources:
 
+  Argo_Climatology:
+    description: "Gridded in-situ temperature and salinity derived from Argo profiles"
+    driver: netcdf
+    args:
+      urlpath: ['/net2/bgr/Data/ARGO/RG_Clim/Updated/RG_ArgoClim_Temp_Extended_200401-202010.nc', '/net2/bgr/Data/ARGO/RG_Clim/Updated/RG_ArgoClim_Psal_Extended_200401-202010.nc']
```

Comment:
point to merged T/S file here: `/archive/Raphael.Dussin/datasets/Roemmich_Gilson_ARGO_TS_Clim`